### DISCUSSION TITLE
Publish an award notice without having previously published it in competition

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#933] [Q&A] [answered]
Hello,

Can an award notice (for example subtype 29) without having previously published it in competition (for example subtype 16)?

In the preview environment it has not been rejected.

With the old publications (before eForms) it did not allow it, we had to include a validation in our platform to not allow it.

In the production environment (with manual validations) can it be rejected?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
It shouldn't currently be the case and controls will only take place after complex rules will get implemented.
